qt/aqt/addcards.py

```diff
@@ -112,6 +115,12 @@ def setupButtons(self) -> None:
         self.compat_add_shorcut = QShortcut(QKeySequence("Ctrl+Enter"), self)
         qconnect(self.compat_add_shorcut.activated, self.addButton.click)
         self.addButton.setToolTip(shortcut(tr.adding_add_shortcut_ctrlandenter()))
+
+        # add io button
+        self.ioAddButton = bb.addButton(f"{tr.actions_add()} {downArrow()}", ar)
```

Comment:
You've used a mix of camelCase and snake_case in this file - it would be nice to stick to snake_case for newly-added items. I do realise it's an easy mistake to make, as Qt uses camelCase, and our legacy code does too.